server/src-lib/Hasura/App.hs

```diff
@@ -269,27 +265,47 @@ runHGEServer ServeOptions{..} InitCtx{..} initTime = do
     asyncActionsProcessor (_scrCache cacheRef) _icPgPool _icHttpManager
 
   -- start a background thread to check for updates
-  _updateThread <- C.forkImmortal "checkForUpdates" logger $ liftIO $ 
+  _updateThread <- C.forkImmortal "checkForUpdates" logger $ liftIO $
```

Comment:
Aside: These trailing whitespace changes are a bummer, sorry. I've tried a plugin in vim that removed them but it ended up being really annoying for various reasons. I should try again. But It's not clear to me why the problem is trailing whitespace and not editors and tools that make a big deal of trailing whitespace...